src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortFilterRecordUnitTest.java

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.GATKBaseTest;
+
+import org.broadinstitute.hellbender.utils.bigquery.AvroFileReader;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ExtractCohortFilterRecordUnitTest extends GATKBaseTest {
+
+    // TODO refactor - this is copied from ExtractCohortRecordUnitTest
+    private static GenericRecord getSingleRecordFromAvroFile (String inputFileName) {
```

Comment:
I want to refactor this but not sure where to put it?

Reply:
I wouldn't worry about it -- it's not much code (it could even be a one liner) and not used from many places